optim.py

```diff
@@ -201,27 +190,30 @@ def forward(self, x, y):
 
     path = ('D:\\Users\\Niels-laptop\\Documents\\2019-2020\\Machine Learning in'
             ' Practice\\Competition 2\\project\\')
-    # path =('/Users/mauriceverbrugge/github/MLiP-2020/kaggle/input/m5-forecasting-accuracy/')
     calendar = pd.read_csv(path + 'calendar.csv')
     prices = pd.read_csv(path + 'sell_prices.csv')
     sales = pd.read_csv(path + 'sales_train_validation.csv')
+    train_sales = pd.concat((sales.iloc[:, :6], sales.iloc[:, 1548:-28]), axis=1)
+    val_sales = pd.concat((sales.iloc[:, :6], sales.iloc[:, -28:]), axis=1)
 
     seq_len = 8
     horizon = 5
+    train_dataset = ForecastDataset(calendar, prices, train_sales, seq_len, horizon)
+    train_loader = DataLoader(train_dataset)
 
-    loader = ForecastDataset(calendar, prices, sales.iloc[:, :100], seq_len=seq_len, horizon=horizon)
-    loader = DataLoader(loader)
+    val_dataset = ForecastDataset(calendar, prices, val_sales)
+    val_loader = DataLoader(val_dataset)
 
-    train_writer = TensorBoardWriter('cpu', True, log_dir='runs/train/')
-    val_writer = TensorBoardWriter('cpu', False, log_dir='runs/val/')
+    train_writer = TensorBoardWriter(log_dir='runs/train/')
+    val_writer = TensorBoardWriter(eval_freq=1, log_dir='runs/val/')
```

Comment:
ik heb start voor een main.py gemaakt, deze strings zouden daar in de toekomst in kunnen. de eerste competitie